test/new-e2e/tests/agent-platform/rpm/rpm_test.go

```diff
@@ -0,0 +1,151 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
🤷 I just used the copyright header snippet included in the `.vscode/` directory, happy to bump the date - I'm not sure what the correct thing to follow is.

Reply:
To be honest I'm not sure what's right either... I would probably have ignored it had I not implemented a "copyright validator" at some point for integrations-core. So do ignore this (since it looks like all files in the repo have the same year).